service/worker/scheduler/workflow.go

```diff
@@ -1234,15 +1235,18 @@ func (s *scheduler) processBuffer() bool {
 			tryAgain = true
 			continue
 		}
-		result, err := s.startWorkflow(start, req)
 		metricsWithTag := s.metrics.WithTags(map[string]string{
 			metrics.ScheduleActionTypeTag: metrics.ScheduleActionStartWorkflow})
+		metricsWithTag.Counter(metrics.ScheduleActionAttempt.Name()).Inc(1)
```

Comment:
I was thinking that instead of a ns-level metric, if it hits this situation it could increment a counter in the ScheduleInfo (they're basically schedule-level metrics). I'm not totally sure which is most valuable tbh. The ScheduleInfo thing is more useful if you're looking at an individual schedule that hasn't run, since it gives a clue why. The ns-level metric would hint at an overall problem but require more digging

Reply:
This case covers the situation when workflow is running, and we try to start the same one again. 
If ScheduleINfo is more useful if you're looking at an individual schedule that hasn't run, that this probably is not the case. 
But I don't have an opinion.

Reply:
The usual situation where this happens is when there are different schedules that start workflows with the same workflow id prefix, at the same time. And then someone wonders why one of them hasn't run (because another one grabbed that id instead). I'm not sure if that helps point in one direction or another...